README.md

```diff
@@ -1151,13 +1151,14 @@ Bear in mind that with the introduction of the new V8 engine alongside the new E
 
 <br /><br /><br />
 
-## ![✔] 8.8. Use multistage builds
+## ![✔] 8.8. Use multi-stage builds
 
-**TL;DR:**
+**TL;DR:** A lot of build-time dependencies and files are not needed for running your application. With multi-stage builds, these resources can be used during
+build, while the runtime environment contains only what's necessary.
```

Comment:
Again, I would say something about WHY is this important. This is also called the 'so what' part